packages/aws-cdk-lib/custom-resources/README.md

```diff
@@ -904,3 +906,20 @@ new s3deploy.BucketDeployment(nestedStackB, "s3deployB", {
     logRetention: logs.RetentionDays.ONE_DAY, // overridden by the `TEN_YEARS` set by `CustomResourceConfig`.
 });
 ```
+
+The `addLogRetentionLifetime` method of `CustomResourceConfig` will associate a logGroup with the lambda. The `dropSpam` lambda stops processing messages that have at least one spam indicator.
+The `addRemovalPolicy` method will configure the log groups removal policy to `DESTROY`.
+```ts
+import * as cdk from 'aws-cdk-lib';
+import * as ses from 'aws-cdk-lib/aws-ses';
+import { CustomResourceConfig } from 'aws-cdk-lib/custom-resources';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'Stack');
+CustomResourceConfig.of(this).addLogRetentionLifetime(logs.RetentionDays.TEN_YEARS);
+CustomResourceConfig.of(app).addRemovalPolicy(cdk.RemovalPolicy.DESTROY);
+
+new ses.ReceiptRuleSet(this, 'RuleSet', {
```

Comment:
Please add some text explaining that this ses thing has a log group because its a custom resource and we called `addLogRetentionLifetime`.